packages/lesswrong/server/apolloServer.ts

```diff
@@ -342,7 +342,7 @@ export function startWebserver() {
     response.setHeader("Content-Type", "text/html; charset=utf-8"); // allows compression
 
     const {bundleHash} = getClientBundle();
-    const clientScript = `<script defer src="/js/bundle.js?hash=${bundleHash}"></script>`
+    const clientScript = `<script src="/js/bundle.js?hash=${bundleHash}"></script>`
```

Comment:
I want to double check whether this gets waited on in any way before merging, we might need to add an `async` tag

Reply:
I think if you have neither `defer` nor `async` here, it blocks rendering entirely until the bundle is fully downloaded. So I would predict this change makes time-to-first-paint a lot worse, at least in the case where the client is bandwidth-bottlenecked.

Reply:
I'm not sure about this behavior but that was my understanding when I put this `defer` in in the first place. This doesn't show up in the profiler results you screenshotted because you're either on localhost or on a fast internet connection, so the bundle was fully downloaded while the serve was SSRing.

Reply:
> I think if you have neither defer nor async here, it blocks rendering entirely until the bundle is fully downloaded

There is conflicting info on this but it looks like this isn't the case and it behaves as `async` by default if it's a dynamically loaded script (from the actual behaviour, and also [this reference](https://javascript.info/script-async-defer#dynamic-scripts) which says so). Nevertheless if it is behaving as async there shouldn't be any harm in adding it, maybe it will prevent issues on other browsers